### DISCUSSION TITLE
Questions about Pockethost!

### DISCUSSION THREAD

**Discussion by ghost** [pockethost/pockethost#171] [Q&A]
Hi,

I don't see a price list on the Pockethost website. How much is it, and if there's a free tier what are the restrictions?

Also, how stable is Pockethost?

Lastly, is our data safely stored with Pockethost?

Thank you.

**Comment by benallfree** [comment]
Hello @basaniteswastaken,

> How much is it, and if there's a free tier what are the restrictions?

No cost and no restrictions at present. When we reach 1.0, we will likely introduce a free tier of something like 100 "run time" minutes per month. This should be enough to cover most hobby and low-traffic projects, but it would not be enough for a realtime application. Realtime applications require the PocketBase instance to run continuously, and for that I'm trying to aim for $20/mo per instance where realtime capability is a must-have. #44 has more info about these plans.

> Also, how stable is Pockethost?

Pretty stable, although #169 and #168 and #165 document several related outages which have been addressed with today's internal updates. 

Background info: something is happening where the child PocketBase processes are not exiting when the Node process restarts. That causes the PocketBase processes to become orphaned and eventually the server runs out of memory. One way (a band-aid) to address this is to make sure the Node process doesn't crash. That's ideal, but of course, bugs will happen and that's why things like pm2 exist. Until I understand why the child processes become orphaned, I consider this to be a serious stability issue and it's at the top of my list of things to solve. 

That said, PocketHost has been running for months and this is the first time we've experienced a bad run of stability issues. Certain enthusiastic users have been helping greatly by reporting issues as they find them. This tells me that people are definitely developing and running production applications on PocketHost, and that's pretty cool!

> is our data safely stored with Pockethost?

It's as safe as it is on your own server. Probably safer, only because the infrastructure is tested and stressed a lot daily. The only way to access the server is via SSH with RSA-2048 encryption. The data on the volume is NOT encrypted at an operating system level (see #143 for discussion), but the VPS itself is of course encrypted by Digital Ocean. 

You can also back up and download all your data at any time. Access your own data via SFTP using your PocketHost account login. 

These are great questions, I'll probably add them to the FAQ page #37.

**Reply by ghost** [reply]
Wow, that was a very quick reply. Thanks for that!

Today I thought of another question though: Are we allowed to have multiple projects running on Pockethost? I already have a server running with Pocketbase, and the only reason I'd want to use Pockethost is if I can run multiple projects at once, and possibly use it for other people's websites that I manage.

**Reply by benallfree** [reply]
You are allowed to run multiple projects. That is actually why I created PocketHost in the first place. I wanted to be able to launch multiple personal projects (web games) and didn't want to create new servers every time. Another good question for the FAQ!

**Reply by ghost** [reply]
Thanks! Can you add a feature to Pockethost for users to delete their projects? I like experimenting with different things, and I don't want to have old discarded projects still taking up space on your server.

**Reply by benallfree** [reply]
#35 and #115 cover what is planned for deleting instances.
